ansible/templates/safegraph-params-prod.json.j2

```diff
@@ -14,7 +14,31 @@
     "sync": true,
     "wip_signal" : []
   },
-  "archive": {
```

Comment:
Why does the archive parameter get unset?

Reply:
As far as I can tell it is not being used and when we get the indicator runner up and running it will trigger running the archiver before it is configured.

Reply:
confirmed; there is no safegraph prefix in the S3 bucket we use for archiving.

Reply:
Thanks.